apps/modernization-api/src/main/java/gov/cdc/nbs/patient/PatientService.java

```diff
@@ -80,31 +73,43 @@ public class PatientService {
 
     @PersistenceContext
     private final EntityManager entityManager;
+    @Autowired
     private final PersonRepository personRepository;
+    @Autowired
     private final CriteriaBuilderFactory criteriaBuilderFactory;
+    @Autowired
     private final ElasticsearchOperations operations;
+    @Autowired
     private final PatientEventRequester requester;
+    @Autowired
     private final UserService userService;
+    @Autowired
     private final PatientLocalIdentifierResolver resolver;
+    @Autowired
+    private final CountryCodeRepository countryCodeRepository;
+    @Autowired
+    private EntityLocatorParticipationRepository entityLocatorParticipationRepository;
+
+
 
     private <T> BlazeJPAQuery<T> applySort(BlazeJPAQuery<T> query, Sort sort) {
         var person = QPerson.person;
         if (sort == null) {
             // if no sort provided, default sort by lastNm then Id
             return query.orderBy(person.lastNm.asc().nullsLast())
-                .orderBy(person.id.desc().nullsLast());
```

Comment:
I love standardizing on one code format but it seems we all have a separate one.  I think @mpeels did some work on this so we'd pull the same params.  It would make it easier to diff.